go.mod

```diff
@@ -1,14 +1,83 @@
 module github.com/pfnet-research/node-operation-controller
 
-go 1.16
+go 1.19
```

Comment:
go 1.18 would be safer? it is because k8s 1.24.x is built on go 1.18.x. I'm unsure how the go version in go.mod affects it.

Reply:
I resolve this because I think it's a minor issue and test passes.